locallibrary/catalog/models.py

```diff
@@ -1,6 +1,8 @@
 from django.db import models
 from django.urls import reverse
 import uuid
+from django.contrib.auth.models import User
+from datetime import date
```

Comment:
thứ tự module django ở sau module python nha
```suggestion
import uuid
from datetime import date
from django.db import models
from django.urls import reverse
from django.contrib.auth.models import User
```

Reply:
Dạ e fix lại r ạ